drawing.py

```diff
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import sys
```

Comment:
Why did these come in? I don't see where they are used

Reply:
Good call, it got introduced in 5973ca76 with my first approach of handling the pyplot issue by checking for `sys.platform == linux`, but then I forgot to remove it when I switched to a different approach.  Will fix.